kubetest/e2e.go

```diff
@@ -319,7 +319,7 @@ func run(deploy deployer, o options) error {
 		}))
 	}
 
-	if o.postTestCmd != "" {
+	if o.preTestCmd != "" && o.postTestCmd != "" {
```

Comment:
Actually since `preTestCmd` is only ran if `o.test` or `o.testCmd` maybe we should add it here too?

Reply:
```suggestion
	if o.postTestCmd != "" && (o.test || (kubemarkUpErr == nil && o.testCmd != "")) {
```